xml2json.py

```diff
@@ -85,20 +85,34 @@ def __init__(self, node):
                     if namesplit[0] in [
                         'definition',
                         'notes',
-                        'search_tags',
+                        'search', # 'search_tags'
```

Comment:
Instead of `split('_')`, it'd be cleaner if we had an explicit list of locales and stripped them off the end. That way `zh_HK` and `search_tags` don't have to be treated differently. In the future, we might add another locale with a sublocale, or another column with an underscore, and forget that we need to special-case them here. (But that probably won't happen for a long time.)